keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -218,6 +218,11 @@ def _forward(self, images, training=None):
         return box_pred, cls_pred
 
     def call(self, images, training=None):
+        if isinstance(images, tf.RaggedTensor):
```

Comment:
Can we add in the test an expected failed test for the ragged input? You could use the same decorator as in:
https://github.com/keras-team/keras-cv/blob/32701437b713050de951351529335f1fc725ceb0/keras_cv/layers/object_detection/multi_class_non_max_suppression_test.py#L50

Reply:
Yeah I will Add tests!

Reply:
Where exactly do I need to add the test?

Reply:
Yes like this:
https://github.com/keras-team/keras-cv/blob/d7fe322699275c965fc25e63bc0fa70c50f1b225/keras_cv/models/object_detection/retina_net/retina_net_test.py#L57